x-pack/test/alerting_api_integration/security_and_spaces/tests/index.ts

```diff
@@ -58,10 +58,14 @@ export async function tearDown(getService: FtrProviderContext['getService']) {
 // eslint-disable-next-line import/no-default-export
 export default function alertingApiIntegrationTests({ loadTestFile }: FtrProviderContext) {
   describe('alerting api integration security and spaces enabled', function () {
-    this.tags('ciGroup17');
+    describe('', function () {
+      this.tags('ciGroup17');
+      loadTestFile(require.resolve('./telemetry'));
+      loadTestFile(require.resolve('./actions'));
+    });
 
-    loadTestFile(require.resolve('./telemetry'));
-    loadTestFile(require.resolve('./actions'));
-    loadTestFile(require.resolve('./alerting'));
+    describe('', function () {
+      loadTestFile(require.resolve('./alerting'));
```

Comment:
Does this need to be tagged with a cigroup?

Reply:
Nope, they are defined inside that file (also in this PR)